ipfx/epochs.py

```diff
@@ -97,7 +97,7 @@ def get_stim_epoch_B(i,test_pulse=True):
     return start_idx, end_idx
 
 
-def get_experiment_epoch(i,v,hz):
+def get_experiment_epoch(i,hz):
     """
     Find index range for the experiment epoch.
     The start index of the experiment epoch is defined as stim_start_idx - PRESTIM_DURATION*sampling_rate
```

Comment:
It would be nice if many of these function docstrings had descriptions for the parameters. (i.e. like the docstring you have on [line 117](https://github.com/AllenInstitute/ipfx/pull/160/commits/d450a2d8f83929907abb760477811e5f8bbad0e7#diff-70ef283edf9491810282fa9533af83e2R117))

Reply:
Thanks, I have added docstings to some of the functions that I expect to stay long term.